packages/splitter/src/useSplitter.ts

```diff
@@ -101,54 +115,82 @@ export function useSplitter({
   defaultValueNow = min,
   valueNow,
   onChange = () => undefined,
-  rtl = false,
+  placement,
   ...props
 }: IUseSplitterProps): IUseSplitterReturnValue {
+  const primaryPaneId = useId();
   const isControlled = valueNow !== undefined && valueNow !== null;
   const [state, setState] = useState(defaultValueNow);
+  const offsetRef = useRef<{ left: number; right: number; top: number; bottom: number }>({
+    left: 0,
+    right: 0,
+    top: 0,
+    bottom: 0
+  });
   const separatorRef = useRef<HTMLElement>(null);
-  const dimension = isControlled ? valueNow : state;
-  const setDimension = isControlled ? onChange : setState;
+  const separatorPosition = isControlled ? valueNow : state;
+  const setSeparatorPosition = isControlled ? onChange : setState;
 
-  const setBoundedDimension = useCallback(
+  const setRangedSeparatorPosition = useCallback(
```

Comment:
Refactored `setBoundedDimension` to `setRangedSeparatorPosition` for more clarity.